common/buildcraft/lib/dimension/FakeWorldServer.java

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (c) 2017 SpaceToad and the BuildCraft team
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+ */
+
+package buildcraft.lib.dimension;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+import javax.annotation.Nullable;
+
+import com.mojang.authlib.GameProfile;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.MoverType;
+import net.minecraft.entity.item.EntityItem;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.DamageSource;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.WorldServerMulti;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.BiomeProvider;
+import net.minecraft.world.chunk.Chunk;
+
+import net.minecraftforge.common.DimensionManager;
+
+import buildcraft.api.schematics.SchematicBlockContext;
+
+import buildcraft.lib.BCLib;
+import buildcraft.lib.misc.data.Box;
+
+public class FakeWorldServer extends WorldServerMulti implements IFakeWorld {
+    private List<ItemStack> drops = new ArrayList<>();
+    public static FakeWorldServer INSTANCE;
+    private FakeChunkProvider provider;
+    private List<Box> locks = new ArrayList<>();
```

Comment:
If you access this list from (potentially) multiple threads then you're going to need to make it threadsafe.